reps/2023-04-28-remove-algorithms-from-rllib.md

```diff
@@ -0,0 +1,193 @@
+## Summary
+
+We'd like to remove aprox ~20 of the 30 algorithms from RLlib. We are considering doing this because 
+
+1. It would reduce the maintenance burden of RLlib.
+2. Many of these algorithms have successors that are more performant and/or easier to hyperparameter tune.
+3. The proposed algorithms have low ussage by the community according to our ray cluster telemetry readings.
+
+However, we recognize that the value that these algorithms have to the community, and so we are establishing the
+`rayproject/rllib-contrib` repo to house these algorithms. Each deprecated algorithm will have its own directory
+in this repo containing the following:
+
+1. The implementation of the algorithm.
```

Comment:
Please capitalize all beginnings of sentences and add dots at the end for consistency.